### PR TITLE
September Release

### PR DESCRIPTION
**Pull request by sukhada**
Includes the following items:

1. https://github.com/350org/action-tools/issues/44
2. https://github.com/350org/ak_intl_v3/issues/385
3. https://github.com/350org/ak_intl_v3/issues/388
4. https://github.com/350org/ak_intl_v3/issues/381
5. https://github.com/350org/ak_intl_v3/issues/368
6. https://github.com/350org/ak_intl_v3/issues/367

### CODE REVIEW

File: afteraction-share.html [hunk 1]
```diff
@@ -312,6 +349,20 @@ <h2>{{ page.custom_fields.share_slide_title }}</h2>
 			afteractionProgressMeter.fadeIn();
 			$(document).trigger('domUpdated'); // tell other scripts to re-run
 			$("#jump-to-form").remove();
+
+			if (afterActionNextStep === 'scroll-donate') {
```
Comment (cjmabry) [inline afteraction-share.html:353]: Could we add code comments here to explain what's going on?

File: afteraction-share.html [hunk 2]
```diff
@@ -415,39 +470,88 @@ <h2>{{ page.custom_fields.share_slide_title }}</h2>
...
+									ga('send', 'event', 'scrolling-signup', 'afteraction-donate-no', window.location.pathname);
+								}
+							});
+						} else if (afterActionNextStep === 'scroll-donate') {
+							// show the donate ask
+							afteractionShowDonate();
+
+							afteraction.on('click', '.afteraction-donate-ask-no', function(e){
+								// prevent getting redirected to the Get Involved URL
+								e.preventDefault();
```
Comment (cjmabry) [inline afteraction-share.html:513]: Looks like this chunk of code is repeated in above step, any way to make this more DRY?

**Review by cjmabry — APPROVED** [review]
Looks good! A few minor questions.

File: user_update.html [hunk 3]
```diff
@@ -49,7 +54,12 @@ <h3 class="section-header meta margin-bottom-large">{{ user.name }}</h3>
 						})
 					</script>
 					{% include "./state_select.html" %}
-
+					{% case 'email' %}
+					{% if user.stats.donated %}
+						<p class="donor-email-label">We noticed you have previously donated. To update your email address ({{ field.value }}), please contact us at <a href="mailto:donations@350.org">donations@350.org</a> so we can make sure your donation records are moved to your new address.</p>
+					{% else %}
+						{{ field }}
```
[the remark this answers is not in the document]
Comment (sukhada) [reply user_update.html:61]: no, we want to render the input for the field, not just the value (which would just render the user's email address)

File: afteraction-share.html [hunk 4]
```diff
@@ -67,24 +67,45 @@
...
 
 <div id="afteraction">
+	{% if page.custom_fields.after_action_next_step == "scroll-share" %}
 	<div id="afteraction-share" class="section afteraction-panel border-top title-section padding-medium width-narrow text-center bg-{% if page.custom_fields.page_background_color %}{{ page.custom_fields.page_background_color }}{% else %}white{% endif %}" aria-live="polite" style="display:none;">
 		<div id="afteraction-share-inner" class="section-inner"></div>
 	</div>
 	<div id="afteraction-donate" class="section afteraction-panel title-section padding-medium width-narrow text-center bg-{% if page.custom_fields.page_background_color %}{{ page.custom_fields.page_background_color }}{% else %}white{% endif %}-fade" aria-live="polite" style="display:none;">
 		<div id="afteraction-donate-inner" class="section-inner"></div>
 	</div>
+	{% else %}
```
[the remark this answers is not in the document]
Comment (sukhada) [reply afteraction-share.html:101]: If `page-redirect` is set, none of this code is used, and the user is redirected to a separate thanks page. We can update to check for `scroll-donate` specifically, but if we add a new option here, we'd presumably want to reorder the steps for that new option, so I think we'd end up updating the code accordingly.

File: afteraction-share.html (hunk 2 above)
Comment (sukhada) [reply afteraction-share.html:514]: Hmm not sure the best way to do this without a larger refactor - maybe something we can tackle in a future sprint so as not to hold back the release this week?

Comment (rachelhbd) [reply afteraction-share.html:514]: Refactoring later works for me - if we add regional variability and embedded form support in future sprints we're likely to see the logic getting more complicated, so maybe best to consider that together

**Review by rachelhbd — APPROVED** [review]
This looks great Sukhada, thanks!

**Pull request merged**